src/blocks/CaseStudyParallax/index.module.scss

```diff
@@ -0,0 +1,314 @@
+@use '@scss/common' as *;
+
+.cards {
+  position: relative;
+}
+
+.mainGutter {
+  position: relative;
+
+}
+
+.card {
+  position: relative;
+  padding-top:  calc(var(--base) * 4);
+
+  &.isFirst {
+    grid-column: 1/16;
+    grid-row: 1/1;
+  }
+
+  & .media {
+    margin-right: calc(var(--gutter-h) / -1);
+    overflow: hidden;
+
+    img {
+      right: -10px;
```

Comment:
@paulpopus Should be able to use `calc` & `var(--base)` here and still get the same `-10px` value